### ISSUE TITLE
Uncaught TypeError: Cannot read property '_aceSession' of undefined

### ISSUE THREAD

**Issue by dmalan** [cs50/harvard.cs50.droplet#27]
```
Uncaught TypeError: Cannot read property '_aceSession' of undefined
  at a.<anonymous> (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/droplet.js:413:66)
  at a.s.fireEvent (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:31:10953)
  at a.s.redrawMain (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:28:19818)
  at a.s.setTopNubbyStyle (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:28:15705)
  at a.s.resizeNubby (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:28:15766)
  at a.c.Editor.a.resizeBlockMode (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:28:11623)
  at a.s.setEditorState (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:31:12280)
  at https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:31:9591
  at Worker.<anonymous> (https://vfs-gce-usw-126-3.c9.io/vfs/1870955/plugins/9c9UparOvTqzxXPewqsR/harvard.cs50.droplet/lib/droplet/droplet-full.js:31:9303)
```

<img width="1279" alt="screen shot 2017-03-08 at 11 18 43 pm" src="https://cloud.githubusercontent.com/assets/788678/23739838/af11f7fa-0455-11e7-9a09-80418d6d3132.png">

**Comment by dmalan** [comment]
@dabbler0 issue seems to be

```js
findAssociatedTab(droplet.session._aceSession, function(tab) {
```

wherein `droplet.session` can sometimes be `undefined`?

**Comment by dabbler0** [comment]
This should be resolved as of 98c3237961f6069027226cb83413df38c73ec2d4.

**Comment by dmalan** [comment]
Thanks!
